_config/model.yml

```diff
@@ -3,13 +3,13 @@ Name: fluentmodel
 ---
 SilverStripe\ORM\DataObject:
   # See FluentExtension
-  cms_localisation_required: false
-  frontend_publish_required: true
+  cms_localisation_required: any
+  frontend_publish_required: none
```

Comment:
Since this is master (new major version) happy to change default to `fallback` to give the best out of the box for the frontend.